setup.cfg

```diff
@@ -21,12 +21,16 @@ packages = find:
 # as per recommendation here https://hynek.me/articles/testing-packaging/
 package_dir =
     =src
+setup_requires =
+    setuptools>=45
+    setuptools_scm[toml]>=6.2
 install_requires =
     # make sure a python 3.9 compatible numpy is selected
     numpy>=1.19.3
     click
     apischema>=0.17.2
     typing_extensions
+    setuptools_scm[toml]>=6.2
```

Comment:
This is only required at dev time, so can you move this to the dev extra please?